plugins/web-motion/skills/gsap-core/SKILL.md

```
@@ -0,0 +1,313 @@
+
+```javascript
+ease: "power1.out"     // default feel
+ease: "power3.inOut"
+ease: "back.out(1.7)"  // overshoot
+ease: "elastic.out(1, 0.3)"
+ease: "none"           // linear
+```
+
+Built-in eases: base (same as `.out`), `.in`, `.out`, `.inOut` where "power" refers to the strength of the curve (1 is more gradual, 4 is steepest):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)